src/dns-client.c

```diff
@@ -0,0 +1,127 @@
+/*
+ *
+ *  oFono - Open Source Telephony
+ *
+ *  Copyright (C) 2014  Canonical Ltd.
+ *
```

Comment:
Does this file require Chromium copyright too?  I ask as there's a comment later in the file that references 'shill', the replacement for 'flimflam'.